meteor/lib/collections/RundownLayouts.ts

```diff
@@ -108,6 +114,13 @@ export interface DashboardLayoutAdLibRegion extends RundownLayoutAdLibRegion {
 	height: number
 }
 
+export interface DashboardLayoutPartCountdown extends RundownLayoutPieceCountdown {
+	x: number
+	y: number
+	width: number
+	height: number
```

Comment:
`width` & `height` don't seem to do much. Perhaps they could be used as an input to scale the font-size of the text in the panel? Otherwise it feels like they can be removed, since the panel is transparent anyway and just contains a single piece of text?

Reply:
I changed it so that the text is centered horizontally, within panel's `width`, and I replaced `height` with `scale`, which scales the font size.